src/widgets/helper/QColorPicker.cpp

```diff
@@ -0,0 +1,244 @@
+#include "widgets/helper/QColorPicker.hpp"
```

Comment:
Please include the original license from the files these were copied from. explain the difference as you explained it in the header file

Reply:
See 0d51d90.